components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,76 @@
+const reddit = require("../../reddit.app.js");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emits an event each time a new link is added to a subreddit",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "The subreddit you'd like to watch for new links.",
+      default: "redditdev",
```

Comment:
I would go ahead and remove the default here. I thought about it more, and I think it might cause some confusion (we can change the description to clarify).